apps/arweave/src/ar_nonce_limiter_server_worker.erl

```diff
@@ -40,6 +69,17 @@ handle_cast(resolve_raw_peer, #state{ raw_peer = RawPeer } = State) ->
 	case ar_util:safe_parse_peer(RawPeer) of
 		{ok, Peer} ->
 			ar_util:cast_after(15000, self(), resolve_raw_peer),
+			case ets:lookup(ar_nonce_limiter, {vdf_client_raw_peer, RawPeer}) of
```

Comment:
any way we can combine this code with the duplicate code in `resolve_remote_server_raw_peers`?

Reply:
looks good!